.github/dependabot.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 updates:
   - package-ecosystem: "cargo"
-    directory: "/"
+    directory: "**/*"
```

Comment:
Specify all directories from the current layer and below recursively, using globstar, for locations of manifest files